.cm/platform_labels.cm

```diff
@@ -107,10 +109,10 @@ platforms:
       - 'subprojects/internal-integ-testing/'
       - 'subprojects/internal-performance-testing/'
       - 'subprojects/internal-testing/'
-  - cc:
-    name: 'configuration_cache'
+  - configuration:
+    name: 'configuration'
```

Comment:
:question: this seems to be inconsistent with other `core/` labels, like `core_execution` and `core_runtime`.

Reply:
Fixed